### PR TITLE
rojo: init at 7.2.1

### PR DESCRIPTION
**Pull request by wackbyte**
###### Description of changes

[Rojo](https://github.com/rojo-rbx/rojo) is a project management tool for Roblox.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by pmiddend** [comment]
I can confirm it builds and runs on my machine. Nothing to critique from my side.

File: pkgs/development/tools/rojo/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+
```
Comment (AndersonTorres) [inline pkgs/development/tools/rojo/default.nix:5]:
```suggestion
```

File: pkgs/development/tools/rojo/default.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  version = "7.2.1";
+
+  src = fetchFromGitHub {
+    owner = "rojo-rbx";
+    repo = "rojo";
+    rev = "v${version}";
+    sha256 = "sha256-Kmq/lBwayYkFU4mbjExj7M9wpg59OkIiTc+2ZrwpuBc=";
+    fetchSubmodules = true;
+  };
+  cargoSha256 = "sha256-qx6Ja0DMe4cEmDSpovtY9T3+0nJS9XivR92K3UKgacE=";
```
Comment (AndersonTorres) [inline pkgs/development/tools/rojo/default.nix:21]:
```suggestion

  cargoSha256 = "sha256-qx6Ja0DMe4cEmDSpovtY9T3+0nJS9XivR92K3UKgacE=";
```

File: pkgs/development/tools/rojo/default.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ];
+
+  meta = with lib; {
+    description = "Project management tool for Roblox";
+    longDescription = "Rojo is a tool designed to enable Roblox developers to use professional-grade software engineering tools.";
```
Comment (AndersonTorres) [inline pkgs/development/tools/rojo/default.nix:33]: Span the long descriptions to multiple lines, 80-char limit. 
It makes the long description more readable.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/development/tools/rojo/default.nix (hunk 3 above)
Comment (SuperSandro2000) [reply pkgs/development/tools/rojo/default.nix:33]: > 80-char limit.

nixpkgs has no 80 characters line limit and it is stupid in 2022. As long as the line is not very long like ~120-180+ characters and it makes sense to split it up everything is fine.

**Pull request merged**